guides/common/modules/proc_viewing-multiple-content-view-environment-hosts.adoc

```diff
@@ -0,0 +1,47 @@
+[id="Viewing_multiple_content_view_environment_hosts{context}"]
+= Viewing multiple content view environment hosts
+
+Use this procedure to view multiple content view environment activation keys in Hammer CLI
+
+.Prerequisites
+Register a host with the created activation key. 
+For more information, see xref:Using_Activation_Keys_for_Host_Registration_{context}[Using activation keys for host registration].
+
+[NOTE]
+====
+Click the *Insecure* checkbox when registering a host.
+====
+
+.Procedure
+. In the {ProjectwebUI}, navigate to *Hosts* > *All Hosts*.
+. Click the *New UI* button.
+. Click *Manage columns*.
+. Under *Content*, click the *Content view environments* checkbox and click *Save*.
+. The multiple content view environments for the host are displayed under the *Content view environments* column.
+. Click on the content view environments to display details.
+. Click on the vertical ellipsis to the right of the content view environments and select *Change content view environments” from the drop-down menu.
+
+[WARNING]
+====
+This host is associated with multiple content view environments. 
+If you assign a lifecycle environment and content view here, the host will be removed from the other environments.
+====
+
+On the *Hosts Details* page, the *Content view environments* card displays the details of the content view environments. 
```

Comment:
```suggestion
On the *Host Details* page, the *Content view environments* card displays the details of the content view environments. 
```

Reply:
If you want to include a note here, I wouldn't be opposed.
```suggestion
On the *Hosts Details* page, the *Content view environments* card displays the details of the content view environments. 

[WARNING]
====
If you reassign the content view environment of a multi-environment host from this screen, the host's content view environments will be overwritten and it will no longer be a multi-environment host. Instead, reassign multi-environment hosts' content view environments using Hammer.
====
```